### DISCUSSION TITLE
Roadmap Priorities

### DISCUSSION THREAD

**Discussion by HEIC-to-JPEG-Dev** [HEIC-to-JPEG-Dev/FileSyncPlusPlus#7] [Polls]
## 👋 Roadmap
We have a good roadmap of features and capabilities for FileSycToy, but we want to make sure **major** changes are prioritised to what people want, so here's a poll.

If you select "Other", please provide some detail in the comments below so we can discuss as a community.

**Comment by JeffRedd** [comment]
I have a lot to say about this. The Microsoft Store has not had good reviews of FileSyncToy but if you get it right it will be a huge hit, just like the original SyncToy was.  I still use it and refuse to use the backup junk that is on the market today with their proprietary disk imaging software that works sometimes and other times not.  Backups are like Gold to the people who have the need for it. I only want a program that copies files from one drive and copies them, along with all the files and directories to another drive in their native state!  No compression of any kind, you are only asking for trouble with compression because it has the tendency to corrupt files especially if you are synchronizing large amounts of data. I really hopes that project continues to grow and become more refined. I have many suggestions including just the types of files you want, like Freebyte Backup Version 3.0. You should look at this very simple backup software utility to get some good ideas. It is no longer being supported. Anyway, you can choose what kind of files you want. Suppose you only wanted to backup *.doc, *.rtf, *.txt, *.MP3 and *.jpg files and nothing more. With a filter like this you could do it and I think it would be nice addition to the program.

It appears that FileSyncToy is longer being developed or I don't see anymore news of it on GitHub and your internet site (duckheadsoftware. com) doesn't even show FileSyncToy anywhere where I can see. I really hope that work on it continues because there is a huge fan base just waiting for it to be resurrected. Wishing you the best of luck with this program, JeffRedd.

**Comment by LikeScientist** [comment]
After purchasing and using it, apart from the smooth UI and simultaneous synchronization of multiple windows, everything else is not as convenient to use as Microsoft's SyncToy. The worst experience is the transfer speed, which takes a long time to search before starting the transfer, possibly due to the lack of subfolders filtering. I hope that at least when SyncToy users want to find better products, FileSyncToy can become a better alternative.
